backend/dataall/base/utils/api_handler_utils.py

```diff
@@ -24,13 +24,22 @@
 ]
 ENGINE = get_engine(envname=ENVNAME)
 ALLOWED_ORIGINS = os.getenv('ALLOWED_ORIGINS', '*')
+AWS_REGION = os.getenv('AWS_REGION')
 
 
 def redact_creds(event):
-    if 'headers' in event and 'Authorization' in event['headers']:
+    if event.get('headers', {}).get('Authorization'):
```

Comment:
Can anyone please tell why are we redacting these ?

Reply:
user creds information does not need to be logged and is no longer relevant for the remaining request lifecycle - opting to redact that info

Reply:
Instead of redacting , can we just extract useful information and then pass it onto our lambda ?